src/hmr/attach_hmr.ts

```diff
@@ -108,10 +108,20 @@ export function attachHMR(ctx: Context) {
     }
   }
 
+  function hardProjectUpdate(ws_instance?: WebSocket) {
+    ctx.log.info('reload', 'Reloading webpage');
+    devServer.clientSend('reload', undefined, ws_instance);
+  }
+
   // here we recieve an update from client - the entire tree of its modules
   devServer.onClientMessage((event, payload: IClientSummary, ws_instance?: WebSocket) => {
-    if (event === 'summary' && payload.id && tasks[payload.id]) {
-      softProjectUpdate(tasks[payload.id], payload, ws_instance);
+    const task = tasks[payload.id];
+    if (event === 'summary' && payload.id && task) {
+      if (task.action === WatcherAction.HARD_RELOAD_MODULES) {
```

Comment:
Hm, no that's not going to work

Reply:
You need to send an event like this:
https://github.com/fuse-box/fuse-box/blob/master/src/hmr/attach_hmr.ts#L144

Then pick it up here:

https://github.com/fuse-box/fuse-box/blob/master/modules/fuse-box-hot-reload/index.js#L21

The best would be to undo the changes in this block too.

Reply:
```
client.on('reload', () => window.location.reload())
```
something like that ;-)

Reply:
I added that in the last commit. I got confused what hard reloading ment in the context of the watcher. Should the check for hard reloading go in the hmr module or is it fine how it is?

Reply:
well, hard reloading means ignoring the cache in this context. I would undo the changes and made a condition where I pointed out. So instead of sending a request for summary just send an event to reload Done ;-) 5 lines of code ;-)

Reply:
Oh ok